src/quoridor/game/game.go

```diff
@@ -25,8 +96,8 @@ func CreateGame(conf *Configuration) (*Game, error) {
 	lineCenter := (boardSize - 1) / 2
 	pawn := Pawn{Position{0, lineCenter}}	
 	id:= shortuuid.New()
-	game := Game{id, pawn, board}
-	storage.Set(game.Id, game)
+	game := Game{id, pawn, []Fence{}, board}
+	storage.Set(game.ID, game)
```

Comment:
You should not save our game here, but just create it. Your function need to have one responsability only

In general, you create, then save it in your controller (aka handler).